frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_detail/mobile_create_row_field_screen.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/mobile/presentation/database/card/card_property_edit/mobile_field_editor.dart';
+import 'package:appflowy/plugins/database_view/application/field/type_option/type_option_context.dart';
+import 'package:appflowy_backend/protobuf/flowy-database2/field_entities.pb.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class MobileCreateRowFieldScreen extends StatefulWidget {
+  final String viewId;
+  final TypeOptionPB typeOption;
+  static const routeName = '/MobileCreateRowFieldScreen';
+  static const argViewId = 'viewId';
+  static const argTypeOption = 'typeOption';
```

Comment:
Place the properties after the constructor and maintain the same style, like this.

<img width="530" alt="Screenshot 2023-11-09 at 11 39 47" src="https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/6862f812-b1d7-487e-8b8d-1b63f8dbd177">